pyvo/registry/regtap.py

```diff
@@ -496,13 +526,33 @@ def access_url(self):
         """
         # some services declare some data models using multiple
         # identifiers; in this case, we'll get the same access URL
-        # multiple times in here.  Don't be alarmed when that happens:
-        access_urls = list(set(self["access_urls"]))
+        # multiple times in here.  Be cool about that situation:
+        access_urls = list(sorted(set(self["access_urls"])))
+
+        if len(access_urls)==0:
+            raise dalq.DALQueryError(
+                f"The resource {self.ivoid} has no queriable interfaces.")
+
+        elif len(access_urls)>1:
+            warnings.warn(AstropyDeprecationWarning(
+                f"The resource {self.ivoid} has multiple capabilities. "
+                " You should explicitly pick one using get_service. "
+                " Returning some access_url now, but this behaviour "
+                " may change in the future."))
+        return access_urls[0]
+
+    def get_unique_access_url(self):
```

Comment:
I don't think a new method is needed at this point, the ultimate goal would be to eliminate the previous behaviour and raise the exception if the access_url is not unique.